extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * .amp-lightboxed gets added to the element when displayed in the lightbox.
+ * .amp-lightboxed is designed to provide a reasonable default behaviour for
+ * lightboxing of arbitary elements.
+ * Both components and page authors are free to customize the style of the
+ * element when inside the lightbox.
+ * Example:
+ *    .myDiv {
+ *       color: blue;
+ *     }
+ *
+ *     .myDiv.amp-lightboxed {
+ *       color: red;
+ *     }
+ */
+.amp-lightboxed {
+  position: fixed !important;
+  z-index: 2147483645 !important;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+/**
+ * The following rule is needed to reset the stacking context.
+ * By resetting the properties that create new stacking context on the
+ * ancestors of the item being lightboxed, we end up with a single stacking
+ * context and therefore the z-index will be absolute. See https://goo.gl/uqY5CN
+ * Code is copied from Webkit's FullScreen Css https://goo.gl/gjbKoI
+ */
+.-amp-lightboxed-ancestor {
+  z-index: auto !important;
+  position: static !important;
+  opacity: 1 !important;
+  transform: none !important;
+  clip: none !important;
+  transition: none !important;
+  will-change: auto !important;
+  -webkit-mask: none !important;
+  -webkit-filter: none !important;
+  -webkit-box-reflect: none !important;
+  -webkit-perspective: none !important;
+  -webkit-transform-style: flat !important;
+  /* TODO(aghassemi): Add MS and Moz support */
+}
+:root.-amp-lightboxed-ancestor {
+  overflow: hidden !important;
+}
+
+/* Page Mask */
+.-amp-lightbox-viewer-mask {
+  position: fixed !important;
+  z-index: 2147483644 !important;
+  top:0 !important;
+  left: 0 !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  background-image: none !important;
+  opacity: 1 !important;
```

Comment:
If it's for shimming, I think it's a fine start. But I also think we don't really need to worry it right now too much. In a way, having a semi-transparent background is a good reminder that we may need to address shimming here.